src/cli/cli.cpp

```diff
@@ -590,6 +590,34 @@ template <> otError Interpreter::Process<Cmd("ba")>(Arg aArgs[])
                 otBorderAgentSetEphemeralKeyCallback(GetInstancePtr(), nullptr, nullptr);
             }
         }
+        /**
+         * @cli ba ephemeralkey feature (enable, disable)
+         * @code
+         * ba ephemeralkey feature
+         * Enabled
+         * Done
+         * ba ephemeralkey feature enable
+         * Done
+         * @endcode
+         * @par api_copy
+         * #otBorderAgentIsEphemeralKeyFeatureEnabled
+         * #otBorderAgentSetEphemeralKeyFeatureEnabled
+         */
+        else if (aArgs[1] == "feature")
+        {
+            if (aArgs[2].IsEmpty())
```

Comment:
I think we may be able to simplify this using `ProcessEnableDisable()` helper. Something like this:
```c++
   error = ProcessEnableDisable(aArg + 2, otBorderAgentIsEphemeralKeyFeatureEnabled, otBorderAgentSetEphemeralKeyFeatureEnabled);
```